README.md

```diff
@@ -170,6 +170,7 @@ json2ts -i foo.json -o foo.d.ts --style.singleQuote --no-style.semi
 ## Custom schema properties:
 
 - `tsType`: Overrides the type that's generated from the schema. Useful for forcing a type to `any` or when using non-standard JSON schema extensions ([eg](https://github.com/sokra/json-schema-to-typescript/blob/f1f40307cf5efa328522bb1c9ae0b0d9e5f367aa/test/e2e/customType.ts)).
+- `tsName`: Overrides the name of a generated interface from the schema. Useful if two items are generated with similar names.
```

Comment:
Could you call this `x-name`? We'll be renaming `tsType` and `tsEnumNames` soon to match the new convention.